include/seqan3/alphabet/nucleotide/dna15.hpp

```diff
@@ -53,10 +51,14 @@
 namespace seqan3
 {
 
+class rna15;
+
 /*!\brief The 15 letter DNA alphabet, containing all IUPAC smybols minus the gap.
  * \ingroup nucleotide
  * \implements seqan3::nucleotide_concept
  * \implements seqan3::detail::constexpr_alphabet_concept
+ * \implements seqan3::trivially_copyable_concept
+ * \implements seqan3::standard_layout_concept
```

Comment:
shouldn't this be in the `base_class`? But okay, maybe someone who uses the `alphabet_base` or `nucleotide_base` and does stupid stuff could mess this requirement up.

Maybe we should subsume `seqan3::detail::constexpr_alphabet_concept`, `seqan3::trivially_copyable_concept` and `seqan3::standard_layout_concept` as `trivial_alphabet_concept` which I think all of our alphabets satisfy.

Reply:
> shouldn't this be in the base_class?

But the base_class itself doesn't guarantee this, a derived type could always violate this.

> Maybe we should subsume seqan3::detail::constexpr_alphabet_concept, seqan3::trivially_copyable_concept and seqan3::standard_layout_concept as trivial_alphabet_concept which I think all of our alphabets satisfy.

Yes, I thought about this, too. Let's discuss it separately.

Reply:
Cool! Put it to the TODO-List